Domain/Entities/Person.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Domain.Common;
+
+namespace Domain.Entities
+{
+    public class Person : BaseEntity
+    {
+        protected Person(int id) : base(id)
```

Comment:
@christopherjomelztan I have moved the EF Core attributes to the DbContext file using Fluent API for the separation of concerns. All database-related functionalities are contained in the Infrastructure layer. It will be reflected in my next PR.